### PR TITLE
add vision v1p1beta1

### PR DESCRIPTION
**Pull request by pongad**

### CODE REVIEW

**Comment by pongad** [comment]
I'm not sure if this should go in a branch? The build won't work yet because protos aren't published.

File: google-cloud-vision/pom.xml [hunk 1]
```diff
@@ -73,12 +77,23 @@
...
       <artifactId>gax-grpc</artifactId>
       <classifier>testlib</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>commons-lang</groupId>
+      <artifactId>commons-lang</artifactId>
+      <version>2.6</version>
+      <scope>test</scope>
+    </dependency>
```
Comment (garrettjonesgoogle) [inline google-cloud-vision/pom.xml]: We shouldn't add this as a new dependency.

File: google-cloud-vision/src/test/java/com/google/cloud/vision/v1p1beta1/ImageAnnotatorSmokeTest.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import com.google.cloud.vision.v1p1beta1.Feature.Type;
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.apache.commons.lang.builder.ReflectionToStringBuilder;
+import org.apache.commons.lang.builder.ToStringStyle;
+
+@javax.annotation.Generated("by GAPIC")
+public class ImageAnnotatorSmokeTest {
```
Comment (garrettjonesgoogle) [inline google-cloud-vision/src/test/java/com/google/cloud/vision/v1p1beta1/ImageAnnotatorSmokeTest.java]: We shouldn't be checking in smoke tests to google-cloud-java.

**Comment by pongad** [comment]
superseded by https://github.com/GoogleCloudPlatform/google-cloud-java/pull/2676

**Pull request closed without merging**